torch/csrc/jit/passes/quantization.cpp

```diff
@@ -671,6 +676,28 @@ void InsertObserversHelper::addIntermediateValuesToSkipObserver(
   }
 }
 
+void InsertObserversHelper::preprocess(
+    script::Module& module,
+    const std::string& method_name) {
+  script::Method method = module.get_method(method_name);
+  auto graph = method.graph();
+  // To cleanup traced graph
+  ConstantPooling(graph);
+  ConstantPropagation(graph);
+  // must do constant propagation first before replacement
+  replaceConvolutionWithConv2d(graph);
```

Comment:
Do these steps need to happen prior to Conv+ BN fusion? It feels like we should have the passes in the order:
 - PreProcess -> FoldConvBN->DeDupModules->InsertObservers.

Wher